public/locales/ru/translations.json

```diff
@@ -56,48 +56,48 @@
   "Block Back": "Заблокировать в ответ",
   "location": "Место хранения списка",
   "owner": "Владелец",
-  "manager": "Менеджер",
+  "manager": "Управляющий",
```

Comment:
Слово «менеджер» в русском языке отсылает к трудовой деятельности человека, так как прочно ассоциируется с должностью в компании. Использование прямого русского перевода кажется более удачным решением. 

The word «manager» in Russian refers to a person's work activity, as it is strongly associated with a position in a company. Using a direct Russian translation seems to be a better solution.

Reply:
На сайте регистратора ens.domains в локализации используют именно "Менеджер"

Reply:
Отличное замечение! Было глупо для меня не обратить на это внимание, ведь я переводил приложение ENS на русский. Давай оставим "Менеджер". 

Fair point! It would have been stupid of me not to notice this, since I was translating the ENS app into Russian. Let's leave the "Manager".

Reply:
Оу, это круто 😮🫡